File: checkerboard/checkerboard.js
let rows = 8;
let cols = 8;

for (let i = 0; i < rows; i++) {
  let board = "";

  for (let j = 0; j < cols; j++) {
    (j + i) % 2 !== 0 ? (board += "#") : (board += " ");
  }
  
  console.log(board);
}

File: Python/checkerboard/checkerboard.py
def checkerboard(rows, cols):
    the_board = ""
    for i in range(0, rows):
    	for j in range(0, cols):
        	if (j + i) % 2 != 0:
            	the_board += "#"
        	else:
            	the_board += " "
            the_board += "\n"
    return the_board
